README.md

```diff
@@ -63,6 +63,18 @@ modified for specific [OpenLab](https://github.com/theopenlab) CI system deploym
 
 6. Update the log server `fqdn` and host key(`secrets.yaml`) in the jobs config repo (`openlab-zuul-jobs`).
 
+## Deploy a new slave of a master of CI system
```

Comment:
hmm.. what I think is this is a independent functionality, and I believe this will not be used in 99% times. so it is better do not merge this steps to the main steps, to avoid confusion in normal deployment guide. what do you think ?